test/test_other.py

```diff
@@ -2375,19 +2388,24 @@ def test_freetype(self):
     self.do_runf('freetype_test.c', expectedOutput,
                  emcc_args=['--use-port=freetype', '--embed-file', 'LiberationSansBold.ttf'])
 
+  @requires_network
+  @requires_network
```

Comment:
```suggestion
```

Reply:
Looks like this fix was missed before landing?

Reply:
(there are two `@requires_network` for this function)

Reply:
I pushed the submit button too quickly.   Fix was landed directly as a followup.